build.gradle

```diff
@@ -107,7 +112,14 @@ android {
 			buildConfigField "String", "SIMPRINTS_MODULE_ID", /"${simprintsModuleId}"/
 		} else {
 			buildConfigField "String", "SIMPRINTS_MODULE_ID", '"Medic Module ID"'
-		}
+    }
+
+    // Every APK requires a unique version code. 
+    // So when compiling multiple APKS for the different ABIs, use the first digit
+    variant.outputs.each { output ->
+      def versionAugmentation = (output.getFilter(OutputFile.ABI) == 'arm64-v8a') ? 1 : 0;
+      output.versionCodeOverride = variant.versionCode * 10 + versionAugmentation
+    }
```

Comment:
Switching between tabs and spaces is breaking my brain. Could you make this whole file consistent, one way or the other?

Reply:
So updated